CK3AGOT/localization/german/agot_event_localization/agot_filler_events/agot_events_sailing_at_sea_l_german.yml

```diff
@@ -1,135 +1,137 @@
-agot_sailing_at_sea.0012.desc:0 "I commanded my men into defensive positions, ready to fight any who'd swing onto deck, my knights and I weren't about to go down without a fight, especially not against no Pirate!\n\nWe might not have had the numbers, but we had the courage, and we had all the energy we could. \n\nAs the pirates closed in, tossing hooks into the side of our ship, I drew my sword and stood at the rail; eyeing the captain, with my men eying opponents of their own,  ready to face our fates. It was going to be a long, hard fight, but I was ready for it. I was a survivor."
-agot_sailing_at_sea.0012.desc.triedtorun:0 "I commanded the men the best I could, we moved the sails in unison with the winds, tried hard turns, and heading towards a storm, hoping to outrun the pirates, but it was no use. They were gaining on us, and I could hear their whoops and hollers as they closed in.\n\nThe thought of escaping now was doomed. But my knights and I weren't about to go down without a fight. I barked more orders to my crew and knights, preparing them for the battle to come. We might not have had the numbers, but we had the courage. \n\nAs the pirates closed in, I drew my sword and stood at the rail; eyeing the captain, with my men eying opponents of their own,  ready to face our fates. It was going to be a long, hard fight, but I was ready for it. I was a survivor."
-agot_sailing_at_sea.0012.a:0 "In a world without gold, we might've been Heroes!"
-agot_sailing_at_sea.0012.tooltip:0 "While you face off in duel against the captain of the opposing ship, A group of your knights will ALSO engage in a single duel against an attacker each."
-agot_sailing_at_sea.0013.toast.title:0 "Ship Boarding!"
-
-agot_sailing_at_sea.0013.friendlydefendervictory.t:0 "Defending from the attempted boarding, You kill [sc_loser.GetFirstNameNoTooltip] in combat!"
-agot_sailing_at_sea.0013.pirateattackervictory.t:0 "While attempting the boarding, You kill [sc_loser.GetFirstNameNoTooltip] in combat!"
-agot_sailing_at_sea.0013.friendlydefenderloser.t:0 "Defending from the attempted boarding, You are stabbed through the gut by the pirate [sc_victor.GetFirstNameNoTooltip]!"
-agot_sailing_at_sea.0013.pirateattackerloser.t:0 "While attempting the boarding, You are cut down by [sc_victor.GetFirstNameNoTooltip]!"
-agot_sailing_at_sea.0013.friendlyoutcome_toast.t:0 "While fighting, you glance over to see your [knight|E] [sc_victor.GetFirstNameNicknamed] cut down the Pirate [sc_loser.GetFirstNameNoTooltip]!"
-agot_sailing_at_sea.0013.piratewinoutcome_toast.t:0 "While fighting, you glance over to see the Pirate [sc_victor.GetFirstNameNicknamed] cut down your [knight|E] [sc_loser.GetFirstNameNoTooltip]!"
+agot_sailing_at_sea.000444.t:0 "Mundfäule: ein Hoffnungsschimmer"
+agot_sailing_at_sea.000444.a:0 "Die Welt erscheint ein wenig heller."
+agot_sailing_at_sea.000444.a.t:0 "Äh... Hallo Sonnenschein?"
+
+
+agot_sailing_at_sea.0007.t:0 "Fracht an Steuerbord!"
```

Comment:
Wenn im Text Cargo mit Trümmern übersetzt wird würde ich dies auch in der Überschrift mit Trümmer bezeichnen